error[E0599]: no function or associated item named `has_no_effect_on_gecko_scrollbars` found for struct `LonghandIdSet` in the current scope
    --> servo/ports/geckolib/glue.rs:4330:53
     |
4330 |     differing_properties.remove_all(&LonghandIdSet::has_no_effect_on_gecko_scrollbars());
     |                                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ function or associated item not found in `LonghandIdSet`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `geckoservo` due to previous error

--- firefox-107.0/servo/ports/geckolib/glue.rs.old	2022-11-26 00:18:27.068795717 +0000
+++ firefox-107.0/servo/ports/geckolib/glue.rs	2022-11-26 00:19:06.218090943 +0000
@@ -4327,7 +4327,6 @@
     // nsIAnonymousContentCreator::CreateAnonymousContent implementations of
     // ScrollFrameHelper and nsScrollbarFrame to clear the AnonymousContentKey
     // if a non-initial value is used.
-    differing_properties.remove_all(&LonghandIdSet::has_no_effect_on_gecko_scrollbars());
 
     if !differing_properties.is_empty() {
         println_stderr!("Actual style:");
